SimCalorimetry/CastorSim/plugins/CastorDigiProducer.h

```diff
@@ -11,14 +11,14 @@
 #include "Geometry/Records/interface/CaloGeometryRecord.h"
```

Comment:
This file was included in CastorDigiProducer.cc, and it should therefore get removed